soccer/gameplay/root_play.py

```diff
@@ -192,7 +195,7 @@ def goalie_id(self):
 
     @goalie_id.setter
     def goalie_id(self, value):
-        self._goalie_id = None if value == -1 else value
+        self._goalie_id = value
```

Comment:
C++ uses -1 to indicate no goalie is selected because it's stored as an int and -1 is the best "invalid" value to use.  Python has the advantage of having more flexible types so it may be nicer to just keep using `None` in python rather than -1.  Either way will work though.